packages/components/src/accordion/src/Accordion.css

```diff
@@ -53,9 +53,9 @@
 /* TRIGGER | BORDERED */
 .o-ui-accordion-bordered .o-ui-accordion-trigger {
     background-color: var(--o-ui-bg-alias-surface);
-    padding: calc(var(--o-ui-sp-4) - 1px) var(--o-ui-sp-5) var(--o-ui-sp-4) var(--o-ui-sp-6);
+    padding: calc(var(--o-ui-sp-4) - 0.0625rem) var(--o-ui-sp-5) var(--o-ui-sp-4) var(--o-ui-sp-6);
     min-height: var(--o-ui-sz-8);
-    border-top: 1px solid var(--o-ui-b-alias-low-break);
+    border-top: 0.0625rem solid var(--o-ui-b-alias-low-break);
```

Comment:
does it make sense for borders to be in REM? last time you mentionned that border are probably the only one that you would prefer having in px, since you always want the border to be 1px

Reply:
For a border to become 2px you would need a font size of roughly 256px. I decided to go rem's everywhere for consistency sake.